### PR TITLE
SCVI: Screen Variants Support

### PR DESCRIPTION
**Pull request by goreraks**
Read API RS_HDSYS_READ_SC_VARIANT_DB
Delete API RS_HDSYS_DELETE_SC_VARIANT
Unfortunately there are no WRITE APIs available, So had to perform the DB operations directly on the piece list tables.
Also navigation from SE10 is not possible, so no JUMP support for now.
#3172

### CODE REVIEW

**Comment by larshp** [comment]
thanks, 

can you help submitting a PR with a simple SCVI example to  https://github.com/abapGit-tests/SCVI ?

File: src/zcl_abapgit_object_scvi.clas.abap [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    lv_screen_variant = ms_item-obj_name.
+
+    SELECT SINGLE chuser
+    FROM shdsvci
+    INTO rv_user
+    WHERE scvariant = lv_screen_variant.
+    IF sy-subrc <> 0
+    OR rv_user IS INITIAL.
+      rv_user = c_user_unknown.
+    ENDIF. " IF sy-subrc <> 0
```
Comment (larshp) [inline src/zcl_abapgit_object_scvi.clas.abap:41]: please remove the end of line comments with ABAP code, in this example `" IF sy-subrc <> 0`

https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#dont-add-method-signature-and-end-of-comments

File: src/zcl_abapgit_object_scvi.clas.abap [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  METHOD zif_abapgit_object~is_active.
+
+    rv_active = is_active( ).
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_object~is_locked.
+
+    rv_is_locked = abap_false. " ' '
```
Comment (larshp) [inline src/zcl_abapgit_object_scvi.clas.abap:143]: and this type of comment

**Comment by goreraks** [comment]
See latest commit for changes done as per the review.
Also example available in the pull request https://github.com/abapGit-tests/SCVI/pull/1

**Comment by larshp** [comment]
thanks, also need to clear the following fields on serialization,

```
     <CRUSER>GORER</CRUSER>
     <CRDATE>2020-04-19</CRDATE>
     <CHUSER>GORER</CHUSER>
     <CHDATE>2020-04-19</CHDATE>
```

on deserialization they can be updated to the current user

**Comment by goreraks** [comment]
User details are handled. Also see the updated https://github.com/abapGit-tests/SCVI/pull/1.

**Comment by larshp** [comment]
welcome to abapGit as a contributor 🎉

**Pull request merged**